### PR TITLE
Model

### PR DESCRIPTION
**Pull request by bax45**

### CODE REVIEW

**Comment by parksb** [comment]
The conflicts seems to have been resolved incorrectly. If you merge this pull request, some of the main changes will be missing😱

**Comment by bax45** [comment]
i made changes only in spawner code to align models as well as in images and their names and deleted floorB and maybe D. Are we using them? If you want to instantiate models that i added you should change in ReferenceImageLibrary names of the images should be same as models'

File: Assets/Scripts/ARObjectSpawner.cs [hunk 1]
```diff
@@ -77,18 +77,16 @@ private void UpdateSpawned(ARTrackedImage image)
...
-            else
+                spawned.gameObject.transform.rotation = Quaternion.Euler(new Vector3(0, 180f, 0));
+            }if (name == "floor_plan1")
             {
-                spawned.gameObject.transform.rotation = Quaternion.Euler(0, image.transform.rotation.eulerAngles.y, 0);
+                Debug.Log(image.transform.position);
+                spawned.gameObject.transform.rotation = Quaternion.Euler(new Vector3(0, 20f, 0));
             }
 
+
```
Comment (parksb) [inline Assets/Scripts/ARObjectSpawner.cs:89]: I don't understand why this change is needed, the object should rotate according to the image.

And you corrected the angle according to the object, can't you change the default orientation of the prefab itself?

File: Assets/RefImageLib/KakaoTalk_20240427_192344409.jpg
Comment (parksb) [inline Assets/RefImageLib/KakaoTalk_20240427_192344409.jpg:1]: Why not use the FloorB image we already have? If you need to add a new one, I think it's better to rename the file to FloorB.jpg.

File: Assets/RefImageLib/floor_plan1.png
Comment (parksb) [inline Assets/RefImageLib/floor_plan1.png:1]: Isn't this image scoring too low to be used?

File: Assets/RefImageLib/floor_plan2.png
Comment (parksb) [inline Assets/RefImageLib/floor_plan2.png:1]: Why not use the existing image FloorA.jpg for this as well?

**Comment by parksb** [comment]
> i made changes only in spawner code to align models as well as in images and their names and deleted floorB and maybe D. Are we using them? If you want to instantiate models that i added you should change in ReferenceImageLibrary names of the images should be same as models'

I see, I left some comments
